napari/layers/base/base.py

```diff
@@ -1142,7 +1142,7 @@ def _get_value_3d(
             The data value along the supplied ray.
 
         """
-        return None
+        return
```

Comment:
Also, I thought the style recommendation for empty methods with docstring was do have no code in it, am I wrong?

Reply:
Yes, but this function looks like a bug. 

Type annotation do not allow for `None` value, and not all subclass overwrite it. So either it should raise an exception, or the type annotation should be fixed. What did you think?

Reply:
Yah, looks like `None` is ok, also the wrapper `get_value` can return it. So I would update the annotation!

Reply:
I have tried to fix the type annotation, but I do not like the result. It looks like the logic here should be checked and fixed.

Reply:
Do you think we should fix now? In the interesting of moving forward, I would rather open an issue about it but move along with this PR for now.

Reply:
created #5653